litestar/contrib/pydantic/pydantic_schema_plugin.py

```diff
@@ -309,16 +308,15 @@ def for_pydantic_model(cls, field_definition: FieldDefinition, schema_creator: S
         )
         property_fields.update(computed_field_definitions)
 
-        return schema_creator.create_component_schema(
+        schema = schema_creator.create_component_schema(
             field_definition,
             required=sorted(f.name for f in property_fields.values() if f.is_required),
             property_fields=property_fields,
             title=title,
-            examples=(
-                None
-                if example is None
-                else get_formatted_examples(
-                    field_definition, [Example(description=f"Example {field_definition.name} value", value=example)]
-                )
-            ),
         )
+
+        if json_schema_extra:
+            for key, value in json_schema_extra.items():
+                setattr(schema, key, value)
```

Comment:
Actually, I think I need to re-work this. `json_schema_extra` should overwrite the dict-formatted schema, not Litestar's `Schema` object...